cmd/gogitver/cmd/root.go

```diff
@@ -94,9 +96,14 @@ func getBranchSettings(cmd *cobra.Command) *git.BranchSettings {
 
 func runRoot(cmd *cobra.Command, args []string) {
 	r, s := getRepoAndSettings(cmd)
+	v := getBoolFromFlag(cmd, "verbose")
+
+	if v {
+		log.SetFlags(0)
```

Comment:
It surprised me, but I guess it makes sense there [isn't a flags value for all-off](https://godoc.org/log#pkg-constants). Why disable the date/time?

Reply:
I thought it was a bit too verbose at the time I wrote this (last year), but I'm rethinking that now. The difference is this:

```
2020/04/23 09:24:04 Found lightweight tag 0.2.1 for ref refs/tags/0.2.1
2020/04/23 09:24:04 Found lightweight tag 0.2.2 for ref refs/tags/0.2.2
2020/04/23 09:24:04 Found lightweight tag 0.3.0 for ref refs/tags/0.3.0
2020/04/23 09:24:04 Found lightweight tag 0.3.1 for ref refs/tags/0.3.1
2020/04/23 09:24:04 Found lightweight tag 0.4.1 for ref refs/tags/0.4.1
2020/04/23 09:24:04 Found lightweight tag 0.4.3 for ref refs/tags/0.4.3
2020/04/23 09:24:04 Found lightweight tag 0.4.4 for ref refs/tags/0.4.4
2020/04/23 09:24:04 Found lightweight tag 0.5.1 for ref refs/tags/0.5.1
2020/04/23 09:24:04 Found lightweight tag 0.6.0 for ref refs/tags/0.6.0
2020/04/23 09:24:04 Found lightweight tag 0.6.1 for ref refs/tags/0.6.1
2020/04/23 09:24:04 Found lightweight tag 0.6.2 for ref refs/tags/0.6.2
2020/04/23 09:24:04 Found lightweight tag 0.6.3 for ref refs/tags/0.6.3
2020/04/23 09:24:04 Found lightweight tag 0.6.4 for ref refs/tags/0.6.4
2020/04/23 09:24:04 Found lightweight tag 0.7.0 for ref refs/tags/0.7.0
2020/04/23 09:24:04 Found lightweight tag 0.8.0 for ref refs/tags/0.8.0
2020/04/23 09:24:04 Found lightweight tag 0.8.1 for ref refs/tags/0.8.1
2020/04/23 09:24:04 Current branch is master
2020/04/23 09:24:04 [abc381d7c1046f1cbeaf0a67231116ef6dc52693] 0.8.1
2020/04/23 09:24:04 [abc381d7c1046f1cbeaf0a67231116ef6dc52693] 0.8.2
0.8.2
```

Versus

```
Found lightweight tag 0.2.1 for ref refs/tags/0.2.1
Found lightweight tag 0.2.2 for ref refs/tags/0.2.2
Found lightweight tag 0.3.0 for ref refs/tags/0.3.0
Found lightweight tag 0.3.1 for ref refs/tags/0.3.1
Found lightweight tag 0.4.1 for ref refs/tags/0.4.1
Found lightweight tag 0.4.3 for ref refs/tags/0.4.3
Found lightweight tag 0.4.4 for ref refs/tags/0.4.4
Found lightweight tag 0.5.1 for ref refs/tags/0.5.1
Found lightweight tag 0.6.0 for ref refs/tags/0.6.0
Found lightweight tag 0.6.1 for ref refs/tags/0.6.1
Found lightweight tag 0.6.2 for ref refs/tags/0.6.2
Found lightweight tag 0.6.3 for ref refs/tags/0.6.3
Found lightweight tag 0.6.4 for ref refs/tags/0.6.4
Found lightweight tag 0.7.0 for ref refs/tags/0.7.0
Found lightweight tag 0.8.0 for ref refs/tags/0.8.0
Found lightweight tag 0.8.1 for ref refs/tags/0.8.1
Current branch is master
[abc381d7c1046f1cbeaf0a67231116ef6dc52693] 0.8.1
[abc381d7c1046f1cbeaf0a67231116ef6dc52693] 0.8.2
0.8.2
```

The dates are probably fine. Do you have a preference here?

Reply:
Not really, was just curious